_data/payments.yml

```diff
@@ -104,6 +104,13 @@ websites:
       tfa: Yes
       software: Yes
       doc: https://privacy.com/faq
+      
+    - name: Ravelin
+      url: https://www.ravelin.com/
+      img: Ravelin.png
+      tfa: Yes
+      software: Yes
+      doc: https://support.ravelin.com/
```

Comment:
With doc link we mean a link to a specific page explaining how to use 2FA on the site.
A general support page doesn't do it.